osquery/tables/system/darwin/unified_log.mm

```diff
@@ -0,0 +1,213 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <CoreFoundation/CoreFoundation.h>
+#include <Foundation/Foundation.h>
+#include <OSLog/OSLog.h>
+
+#include <osquery/core/tables.h>
+#include <osquery/logger/logger.h>
+
+#include <boost/algorithm/string/replace.hpp>
+
+namespace ba = boost::algorithm;
+
+namespace osquery {
+namespace tables {
+
+const std::map<ConstraintOperator, NSPredicateOperatorType> supportedOps = {
```

Comment:
nitpick, do you mind using the naming pattern: `kSupportedOps` for this and similarly for other constants?